browser.go

```diff
@@ -54,12 +57,21 @@ func (b *Browser) HideLine(line int) {
 }
 */
 
-//TODO: implement
-/*
-func (b *Browser) Icon() {
+func (b *Browser) Icon(line int) Image {
+		fmt.Println(b.icons)
```

Comment:
Is this needed? Looks like a leftover from a debugging.

Reply:
Oops, yeah it is. I'll remove it

Reply:
It's been updated